apps/dashboard/package.json

```diff
@@ -2,7 +2,44 @@
   "name": "openondemand-dashboard",
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.15.4",
+    "@popperjs/core": "^2.11.8",
     "@rails/ujs": "^7.0.1",
+    "@uppy/audio": "^1.1.9",
+    "@uppy/aws-s3": "^3.6.2",
+    "@uppy/aws-s3-multipart": "^3.12.0",
+    "@uppy/box": "^2.4.0",
+    "@uppy/companion-client": "^3.8.1",
+    "@uppy/compressor": "^1.1.4",
+    "@uppy/core": "^3.13.0",
+    "@uppy/dashboard": "^3.9.0",
+    "@uppy/drag-drop": "^3.1.0",
+    "@uppy/drop-target": "^2.1.0",
+    "@uppy/dropbox": "^3.4.0",
+    "@uppy/facebook": "^3.3.1",
+    "@uppy/file-input": "^3.1.2",
+    "@uppy/form": "^3.2.1",
+    "@uppy/golden-retriever": "^3.2.0",
+    "@uppy/google-drive": "^3.6.0",
+    "@uppy/image-editor": "^2.4.6",
+    "@uppy/informer": "^3.1.0",
+    "@uppy/instagram": "^3.3.1",
+    "@uppy/onedrive": "^3.4.0",
+    "@uppy/progress-bar": "^3.1.1",
+    "@uppy/provider-views": "^3.13.0",
+    "@uppy/redux-dev-tools": "^3.0.3",
+    "@uppy/remote-sources": "^1.3.0",
+    "@uppy/screen-capture": "^3.2.0",
+    "@uppy/status-bar": "^3.3.3",
+    "@uppy/store-default": "^3.2.2",
+    "@uppy/store-redux": "^3.0.7",
+    "@uppy/thumbnail-generator": "^3.1.0",
+    "@uppy/transloadit": "^3.8.0",
+    "@uppy/tus": "^3.5.5",
+    "@uppy/unsplash": "^3.3.1",
+    "@uppy/url": "^3.6.1",
+    "@uppy/utils": "^5.9.0",
+    "@uppy/webcam": "^3.4.2",
+    "@uppy/zoom": "^2.3.1",
```

Comment:
Are you sure we need all these? Doesn't the `uppy` package just have all these?

Reply:
I'm not sure if we need them all. The new version of uppy seems to use @uppy/core and each other package specifically, instead of just 'uppy'. i'll try to find and remove the uppy dependencies that weren't used before

Reply:
Yea I would say, let's remove all or start with just `@uppy/core` and then see what we need, which I'm guessing is not many of them.